content/docs/result-components/ReactiveList.md

```diff
@@ -76,6 +76,8 @@ Example uses:
     whether to stream new result updates in the UI. Defaults to `false`. `stream: true` is appended to the streaming hit objects, which can be used to selectively react to streaming changes (eg. showing fade in animation on new streaming hits, Twitter/Facebook like streams, showing the count of new feed items available like *2 New Tweets*)
 - **scrollTarget** `String` [optional]  
     accepts `id` of the container you wish to apply infinite loading on. **Note:** The container should be scrollable.
+- **scrollOnChange** `Boolean` [optional]  
+    prevents scrolling to top whenever the query changes. Defaults to `true`, i.e. scrolls to top of the window or container when the query changes.
```

Comment:
How does it prevent scrolling to top if the default is `true`?

Reply:
Yeah I think we should replace the sentence with `handle scrolling of page whenever the query changes`

Reply:
Enables you to customise the window scrolling experience on query change. Defaults to `true` i.e. The window will scroll to top in case of the query change, which can be triggered by change in pagination, change in filters or search value, etc. When set to `false`, scroll position will stay intact.